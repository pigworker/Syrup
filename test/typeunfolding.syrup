type <Bitt> = <Bit>

nand2(<Bitt>, <Bit>) -> <Bit>
nand2(X, Y) = nand(X,Y)

experiment nand = nand2