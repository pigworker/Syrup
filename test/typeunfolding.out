Type `<Bitt>` is defined.

Circuit `nand2` is defined.

Bisimulation between `nand`, `nand2`:
  `nand` behaves like `nand2`
    {} ~ {}
